Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Use the official Node.js image as the base image
+FROM node:lts as base
```

Comment:
I'd suggest locking the node version here to avoid future disruptions and update manually as needed

Reply:
Also you dont need to declare as a base, because you have a single image to build!

Reply:
@leordev what should this line read as? I'm not sure what the base keyword is doing here